<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>version</class>
 <widget class="QDialog" name="version">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1490</width>
    <height>1029</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_soft_v">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件发布版本：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_machine_type_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>651</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_machine_type">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>仪器规格型号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_soft_v_all_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>310</y>
     <width>661</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_soft_v_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>671</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_serail">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>650</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;序列号：      &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mac_address">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>540</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MAC address：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>770</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/back.png);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_serail_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>660</y>
     <width>511</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_soft_v_all">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>310</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>软件完整版本：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_mac_address_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>521</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
